pkg/vsphere/tasks/waiter.go

```diff
@@ -77,7 +77,7 @@ func WaitForResult(ctx context.Context, f func(context.Context) (ResultWaiter, e
 			cerr = errors.Errorf("%s - (%s)", cerr, info.Error)
 		}
 
-		log.Errorf(cerr.Error())
+		//		log.Errorf(cerr.Error())
```

Comment:
Please make this debug - we can look into package level log configuration later, but this shouldn't be squashed completely for the other components.